src/Controllers/SettingsController.php

```diff
@@ -282,7 +286,7 @@ public function saveIfsgCertificate(Request $request): Response
 
     public function saveDrivingLicense(Request $request): Response
     {
-        if (!$this->checkDrivingLicense()) {
+        if (!($this->checkDrivingLicense() || config('driving_license_enabled'))) {
```

Comment:
```suggestion
        if (!config('driving_license_enabled') && !$this->checkDrivingLicense()) {
```

is cleaner in its intend (and checking the config before querying the db is also faster)